packages/wrangler/src/cli.ts

```diff
@@ -1,5 +1,28 @@
+import * as Sentry from "@sentry/node";
+import process from "process";
 import { hideBin } from "yargs/helpers";
+
 import { main } from ".";
+import * as pkj from "../package.json";
+
+Sentry.init({
+  release: `${pkj.name}@${pkj.version}`,
+  initialScope: {
+    tags: { Wrangler2: pkj.version },
```

Comment:
```suggestion
    tags: { wrangler: pkj.version },
```

Reply:
Could we also add some metadata? OS, architecture, Node version, etc. (Account ID would be great to identify issues as well, but you would need to add that somewhere in `cfetch`, I think)

Reply:
In local changes I had already changed it to be the same `pkj.name` as the release. As for making it `wrangler` wouldn't that make it harder to distinguish from older wrangler when running in the same "Wrangler" project in Sentry. 

As for metadata and any other additional insights, we can definitely look into that 😄